app/views/link_accounts/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :title, t("link_accounts.things_are_changing.heading") %>
+
+<div class="govuk-grid-row">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: t("link_accounts.things_are_changing.heading"),
+      heading_level: 1,
+      font_size: "l",
+      margin_bottom: 6,
+    } %>
+  <%= sanitize(t("link_accounts.things_are_changing.description")) %>
+
+  <%= render 'govuk_publishing_components/components/button', {
+    text: t("link_accounts.things_are_changing.action_button.#{@link_or_sign_in}"),
+    href: process_govuk_account_path,
+    margin_bottom: true,
+    data_attributes: {
+      module: 'auto-track-event',
+      track_category: 'things_are_changing',
+      track_action: t("link_accounts.things_are_changing.tracking.action_button.#{@link_or_sign_in}"),
+    },
+  } %>
```

Comment:
Let's change this to a POST form (and so add a `post` route for the controller method), so we get Rails' CSRF protection.

We still need to add some CSRF protection around authentication (eg, so you can't just send a user to gov.uk/email/manage/authenticate/account` and get their accounts merged) but that can be done in govuk_personalisation I think.

Reply:
Ace, great change, thanks for suggesting.

That's now here in [this commit](https://github.com/alphagov/email-alert-frontend/pull/1107/commits/3af2e841f99a8afc66e1716bc53a622e9dbdcf62), let me know what you think?